logic/src/commonMain/kotlin/com/wire/kalium/logic/featureFlags/KaliumConfigs.kt

```diff
@@ -0,0 +1,44 @@
+package com.wire.kalium.logic.featureFlags
+
+data class KaliumConfigs(
+    val isChangeEmailEnabled: Boolean = false,
+    val isMarketingCommunicationEnabled: Boolean = false,
+    val isSSoEnabled: Boolean = false,
+    val isAccountCreationEnabled: Boolean = false,
+    val blockOnJailbreakOrRoot: Boolean = false,
+    val blockOnPasswordPolicy: Boolean = false,
+    val forceAppLock: Boolean = false,
+    val forceConstantBitrateCalls: Boolean = false,
+    val forceHideScreenContent: Boolean = false,
+    val forcePrivateKeyboard: Boolean = false,
+    val keepWebSocketOn: Boolean = false,
+    val wipeOnCookieInvalid: Boolean = false,
+    val submitCrashReports: Boolean = false,
+    val webLinkPreview: Boolean = false,
+    val developerFeaturesEnabled: Boolean = false,
+    val isLoggingEnabled: Boolean = false,
+    val isSafeLoggingEnabled: Boolean = false,
+    val enableBlacklist: Boolean = false,
+    val fileRestrictionEnabled: Boolean = false,
+    val countlyAppKey: String = "",
+    val countlyServerUrl: String = "",
```

Comment:
Do we still use Countly?

Reply:
I guess not , but I left it there as I parse the values as it is , but I think for now I can remove the things that we know that we won't use now , and can add later